level-2/예상-대진표.js

```diff
@@ -13,3 +13,30 @@ function solution(n, a, b) {
     }
     return currentRound
 }
+
+//정답 2 - le2sky
+function solution(n, a, b) {
+  let arr = Array.from({ length: n }, () => 0)
+  arr[b - 1] = "B"
+  arr[a - 1] = "A"
+
+  const isDiff = () => {
+    return (
+      (arr.indexOf("A") + 1 > arr.length / 2 && arr.indexOf("B") + 1 <= arr.length / 2) ||
+       (arr.indexOf("A") + 1 <= arr.length / 2 && arr.indexOf("B") + 1 > arr.length / 2)) ? true : false
```

Comment:
🌱 반영해도 좋고 넘어가도 좋아요. 🌱

불필요한 tenary operator가 사용되어 가독성이 조금 떨어진다고 생각합니다! boolean 값을 직접 리턴 하시는 것보단 비교문 그 자체를 쓰시는 것이 좋을 것이라 생각됩니다!
```suggestion
    return (
      (arr.indexOf("A") + 1 > arr.length / 2 && arr.indexOf("B") + 1 <= arr.length / 2) ||
       (arr.indexOf("A") + 1 <= arr.length / 2 && arr.indexOf("B") + 1 > arr.length / 2))
```